src/com/google/foam/demos/olympics/Controller.js

```diff
@@ -30,8 +30,13 @@ foam.CLASS({
     'foam.u2.TableView'
   ],
 
+  imports: [
+    'window'
+  ],
+
   constants: {
-    CSS_NAME: 'foam-u2-search-FilterController'
+    CSS_NAME: 'foam-u2-search-FilterController',
```

Comment:
Make a note on the next design meeting agenda for us to discuss this and try to find something cleaner or more predictable.